content/docs/1.18/opentelemetry.md

```diff
@@ -0,0 +1,115 @@
+---
+title: OpenTelemetry
+hasparent: true
+---
+
+The future Jaeger backend components will be based on [OpenTelemetry collector](https://opentelemetry.io/docs/collector/about/).
+This integration will make all OpenTelemetry Collector features available in the Jaeger backend components.
+
+{{< warning >}}
+At the moment Jaeger OpenTelemetry components are experimental and the configuration and behavior can change.
+The current progress can be tracked via [issues tagged as `area/otel`](https://github.com/jaegertracing/jaeger/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fotel).
+{{< /warning >}}
+
+The Jaeger OpenTelemetry backend components are published as Docker images:
+
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/)
+**jaeger-collector**  | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/)
+**jaeger-ingester**   | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/)
+**all-in-one**        | [hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/](https://hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/)
+
+## Compatibility
+
+The Jaeger OpenTelemetry components are **almost** backward compatible legacy Jaeger components.
```

Comment:
"compatible with the legacy" - although should we call them legacy yet? Possibly "current Jaeger components"?